packages/fxa-auth-server/test/local/routes/sign.js

```diff
@@ -35,7 +35,7 @@ describe('/certificate/sign', () => {
     );
     return SessionToken.create({
       uid: uuid.v4({}, Buffer.alloc(16)).toString('hex'),
-      email: 'foo@example.com',
+      email: 'vbudhram+mozilla@mozilla.com',
```

Comment:
Did you mean to leave this here?

Reply:
I can revert this, it was just me testing different values.